### PR TITLE
*: change standard "context" package to "golang.org/x/net/context"

### PR DESCRIPTION
**Pull request by tiancaiamao**
store/tikv should not use both "context" and "golang.org/x/net/context"

### CODE REVIEW

File: distsql/distsql.go [hunk 1]
```diff
@@ -14,11 +14,12 @@
 package distsql
 
 import (
-	goctx "context"
 	"io"
 	"io/ioutil"
 	"time"
 
+	goctx "golang.org/x/net/context"
+
```
Comment (zimulala) [inline distsql/distsql.go]: Remove this empty line.

File: distsql/distsql_test.go [hunk 2]
```diff
@@ -15,14 +15,15 @@ package distsql
...
-	goctx "context"
 	"errors"
 	"io"
 	"io/ioutil"
 	"runtime"
 	"testing"
 	"time"
 
+	goctx "golang.org/x/net/context"
+
```
Comment (zimulala) [inline distsql/distsql_test.go]: Remove this empty line.

File: store/tikv/coprocessor_slow_test.go [hunk 3]
```diff
@@ -16,11 +16,11 @@
 package tikv
 
 import (
-	"context"
+	mocktikv "github.com/pingcap/tidb/store/tikv/mock-tikv"
+	goctx "golang.org/x/net/context"
```
Comment (shenli) [inline store/tikv/coprocessor_slow_test.go]: Remove the empty line.

File: store/tikv/rawkv_test.go [hunk 4]
```diff
@@ -14,7 +14,7 @@
 package tikv
 
 import (
-	"context"
+	goctx "golang.org/x/net/context"
```
Comment (shenli) [inline store/tikv/rawkv_test.go]: Remove the empty line.

File: store/tikv/region_cache.go [hunk 5]
```diff
@@ -15,9 +15,10 @@ package tikv
 
 import (
 	"bytes"
-	goctx "context"
 	"sync"
 
+	goctx "golang.org/x/net/context"
```
Comment (shenli) [inline store/tikv/region_cache.go]: Remove the empty line.

File: store/tikv/store_test.go [hunk 6]
```diff
@@ -14,10 +14,11 @@
 package tikv
 
 import (
-	goctx "context"
 	"sync"
 	"time"
 
+	goctx "golang.org/x/net/context"
+
```
Comment (shenli) [inline store/tikv/store_test.go]: Remove the empty line.

**Comment by siddontang** [comment]
I think we must convert all standard context to golang context now. It is very strange that we use two similar context packages in one project and this it a risk to combine using them together.

**Comment by coocood** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

We can add go 1.6 in travis CI to avoid using wrong context.

**Pull request merged**